README.md

```diff
@@ -2,95 +2,37 @@
-      participant Match as Matchmaker
-      participant Server as Server
-      Client ->> Model: GET Model Y
-      Model ->> Client: Model Y
-      Client ->> Client: Inference
-      par Actualise model
-      Client ->> Model: POST Inference Gradient
-      Model ->> Model: Actualise Model Y
-      and Get UDP server
-      Client ->> Match: POST Set of users
-      Match ->> Match: Match users
-      Match ->> Server: GET UDP Server
-      Server ->> Match: PORT UDP Server
-      Match ->> Client: PORT UDP Server
-      end
-```
-
-In a **distributed architecture**, the code that infers the correct set of users belongs to the **client app**. Thus, it uses the client technologies: Godot and GDScript.
-
-## Centralised
+In a **centralised architecture**, the code that infers the correct set of users belongs to the **cloud**. Thus, it uses whatever languages.
 
-```mermaid
-sequenceDiagram
-      participant Client as Client
-      participant Model as Model provider
-      participant Match as Matchmaker
-      participant Server as Server
-      Client ->> Model: GET Set of users
-      par Client model
-      Model ->> Model: Get Model X of Client
-      and Global model
-      Model ->> Model: Get Model Y
-      end
-      Model ->> Model: Inference
-      Model ->> Model: Actualise Model Y
-      Model ->> Client: Set of users
-      Client ->> Match: POST Set of users
-      Match ->> Match: Match users
-      Match ->> Server: GET UDP Server
-      Server ->> Match: PORT UDP Server
-      Match ->> Client: PORT UDP Server
-```
+## Ideal Architecture - Distributed and Decentralised
 
-In a **centralised architecture**, the code that infers the correct set of users belongs to the **cloud**. Thus, it uses whatever languages.
+> `Federated` architecture will simply remove the **_client models_ database**
 
-> We will first choose the **centralised architecture** to use state-of-the-art libraries in Machine Learning with **Python**. However, we'll keep the code as close as possible to a **decentralised** version, so we will be able to switch later on. The objective is to verify as quickly as possible that the model works.
+In a **distributed architecture**, the code that infers the correct set of users belongs to the **user app**. Thus, it uses the client technologies.
 
 # How an user find a room?
 
 ```mermaid
 sequenceDiagram
-    participant RS as Recommender system
-    actor User
-    participant MM as Matchmaker
-    participant FM as Fleet manager
-    User ->> RS: who are the best users for me?
-    RS ->> User: Set of users
-    User ->> MM: I want a room with these users
+    participant RS as recommender system
+    actor u as user
+    participant MM as matchmaker
+    participant FM as fleet manager
+    u ->> RS: who are the best users for me?
+    RS ->> u: Set of users
+    u ->> MM: I want a room with these users
     MM ->> MM: find best rooms
     MM -->> FM : create new rooms
     FM -->> MM : endpoints
```

Comment:
I would say 'Create or Update rooms', since it will not always be a room creation ?